app/ante_no_seq.go

```diff
@@ -59,12 +74,21 @@ func NewAnteHandler(
 		ante.TxTimeoutHeightDecorator{},
 		ante.NewValidateMemoDecorator(ak),
 		ante.NewConsumeGasForTxSizeDecorator(ak),
-		deductFeeDecorator,
+		block.NewIgnoreDecorator(
+			deductFeeDecorator,
+			blockSDKParams.freeLane,
+		),
 		ante.NewSetPubKeyDecorator(ak), // SetPubKeyDecorator must be called before all signature verification decorators
 		ante.NewValidateSigCountDecorator(ak),
 		ante.NewSigGasConsumeDecorator(ak, sigGasConsumer),
-		ante.NewSigVerificationDecorator(ak, signModeHandler),
-		// ante.NewIncrementSequenceDecorator(ak),
+		// ante.NewSigVerificationDecorator(ak, signModeHandler) <-- removed this to prevent failures resulting from invalid tx orders in e2e
+		ante.NewIncrementSequenceDecorator(ak),
```

Comment:
This looks weird to me. Seems like we should end up w/ SigVerificationDecorator enabled right?

Reply:
The reason we ignore sig verification is that it's possible that the sequence # of txs is incorrect, the sigVerificationDecorator is where this is checked





[3:00](https://skip-protocol.slack.com/archives/D067ERWBXR9/p1705618823070209)
here: https://github.com/cosmos/cosmos-sdk/blob/03d578b9de5136ffc658f2b0c07434373b07b7da/x/auth/ante/sigverify.go#L268